db_stress_tool/db_stress_shared_state.h

```diff
@@ -260,6 +266,16 @@ class SharedState {
     return expected_state_manager_->ClearColumnFamily(cf);
   }
 
+  void SetPersistedSeqno(SequenceNumber seqno) {
+    MutexLock l(&persist_seqno_mu_);
+    return expected_state_manager_->SetPersistedSeqno(seqno);
+  }
+
+  SequenceNumber GetPersistedSeqno() {
+    MutexLock l(&persist_seqno_mu_);
+    return expected_state_manager_->GetPersistedSeqno();
+  }
+
```

Comment:
@archang19  I missed the lock here sorry! We can't use the existing SharedState::mutex_ since some flush will happens within SharedState::mutex_ being held hence deadlock

Reply:
Okay, I don't understand what is going on in the code snippet you linked, but the new approach with `persist_seqno_mu_` lgtm